copter/source/docs/tuning-process-instructions.rst

```diff
@@ -158,96 +286,95 @@ If the aircraft appears stable enough to attempt autotune follow the instruction
+- :ref:`ATC_RAT_YAW_I <ATC_RAT_YAW_I__AC_AttitudeControl_Multi>` = :ref:`ATC_RAT_YAW_P <ATC_RAT_YAW_P__AC_AttitudeControl_Multi>` x 0.1
 
 These values should only be changed if Autotune produces higher values. Small aerobatic aircraft may prefer to keep these values as high as possible.
 
 Setting the input shaping parameters
 ------------------------------------
 
-Arducopter has a set of parameters that define the way the aircraft feels to fly. This lets allows the aircraft to be set up with a very aggressive tune but still feel like a very docile and friendly aircraft to fly. 
+Arducopter has a set of parameters that define the way the aircraft feels to fly. This lets allows the aircraft to be set up with a very aggressive tune but still feel like a very docile and friendly aircraft to fly.
 
 The most important of these parameters is:
 
-- ACRO_YAW_P - yaw rate x 45 degrees/s
-- ANGLE_MAX -  maximum lean angle
-- ATC_ACCEL_P_MAX - Pitch rate acceleration
-- ATC_ACCEL_R_MAX - Roll rate acceleration
-- ATC_ACCEL_Y_MAX - Yaw rate acceleration
-- ATC_ANG_LIM_TC - Aircraft smoothing time
+- :ref:`ACRO_YAW_P <ACRO_YAW_P>` - yaw rate x 45 degrees/s
+- :ref:`ANGLE_MAX <ANGLE_MAX>` -  maximum lean angle
+- :ref:`ATC_ACCEL_P_MAX <ATC_ACCEL_P_MAX>` - Pitch rate acceleration
+- :ref:`ATC_ACCEL_R_MAX <ATC_ACCEL_R_MAX>` - Roll rate acceleration
+- :ref:`ATC_ACCEL_Y_MAX <ATC_ACCEL_Y_MAX>` - Yaw rate acceleration
+- :ref:`ATC_ANG_LIM_TC <ATC_ANG_LIM_TC>` - Aircraft smoothing time
 
-Autotune will set the ATC_ACCEL_X_MAX parameters to their maximum based on measurements done during the Autotune tests. These values should not be increased beyond what Autotune suggests without careful testing. In most cases pilots will want to reduce these values significantly.
+Autotune will set the :ref:`ATC_ACCEL_X_MAX <ATC_ACCEL_X_MAX>` parameters to their maximum based on measurements done during the Autotune tests. These values should not be increased beyond what Autotune suggests without careful testing. In most cases pilots will want to reduce these values significantly.
 
-For aircraft designed to carry large directly mounted payloads, the maximum values of ATC_ACCEL_X_MAX should be reduced based on the minimum and maximum take off weight (TOW):
-ATC_ACCEL_X_MAX  x (min_TOW / max_TOW)
+For aircraft designed to carry large directly mounted payloads, the maximum values of :ref:`ATC_ACCEL_X_MAX <ATC_ACCEL_X_MAX>` should be reduced based on the minimum and maximum take off weight (TOW):
+:ref:`ATC_ACCEL_X_MAX <ATC_ACCEL_X_MAX>`  x (min_TOW / max_TOW)
 
-ACRO_YAW_P should be set to be approximately 0.5 x ATC_ACCEL_Y_MAX / 4500 to ensure that the aircraft will get upto full yaw rate and stop again in approximately half a second.
+:ref:`ACRO_YAW_P <ACRO_YAW_P>` should be set to be approximately 0.5 x :ref:`ATC_ACCEL_Y_MAX <ATC_ACCEL_Y_MAX>` / 4500 to ensure that the aircraft will get up to full yaw rate and stop again in approximately half a second.
 
```

Comment:
to ensure that the aircraft can achieve full yaw rate in approximately half a second.